frontend/src/css/style.css

```
@@ -0,0 +1,23 @@
+
+
+@font-face {
+    font-family: Ubuntu;
+    src: url("../../public/assets/fonts/Ubuntu-Regular.ttf");
+}
+
+@font-face {
+    font-family: Mpulus;
+    src: url("../../public/assets/fonts/MPLUSRounded1c-Regular.ttf");
+}
+@font-face {
+    font-family: Josefin;
+    src: url("../../public/assets/fonts/JosefinSans-Regular.ttf");
+}
```

実行すると以下の警告が出るので，フォントファイルのパスを変更した方がよさそう

```
Files in the public directory are served at the root path.
Instead of /public/assets/fonts/JosefinSans-Regular.ttf, use /assets/fonts/JosefinSans-Regular.ttf.
Files in the public directory are served at the root path.
Instead of /public/assets/fonts/JosefinSans-Regular.ttf, use /assets/fonts/JosefinSans-Regular.ttf. (x2)
Files in the public directory are served at the root path.
Instead of /public/assets/fonts/MPLUSRounded1c-Regular.ttf, use /assets/fonts/MPLUSRounded1c-Regular.ttf.
Files in the public directory are served at the root path.
Instead of /public/assets/fonts/Ubuntu-Regular.ttf, use /assets/fonts/Ubuntu-Regular.ttf.
```

```suggestion
@font-face {
    font-family: Ubuntu;
    src: url("/assets/fonts/Ubuntu-Regular.ttf");
}

@font-face {
    font-family: Mpulus;
    src: url("/assets/fonts/MPLUSRounded1c-Regular.ttf");
}
@font-face {
    font-family: Josefin;
    src: url("/assets/fonts/JosefinSans-Regular.ttf");
}
```

---

直した